src/vs/workbench/services/files/electron-browser/fileService.ts

```diff
@@ -26,6 +26,7 @@ import { IEnvironmentService } from 'vs/platform/environment/common/environment'
 import { IEditorGroupService } from 'vs/workbench/services/group/common/groupService';
```

Comment:
Pending response in https://github.com/Microsoft/vscode/pull/12400#pullrequestreview-4093616